docs/user/private-app-repository.md

```diff
@@ -57,6 +58,46 @@ It is possible to configure ChartMuseum to use authentication with two different
   - Set as Authorization Header a `Basic` auth header using as value the base64 codification of the string `user:password`. For example, for the user "Aladdin" and password "open sesame", it would use the following header field: `Basic QWxhZGRpbjpvcGVuIHNlc2FtZQ==`
 - Using a [JWT token](https://github.com/chartmuseum/auth-server-example). Once you obtain a valid token you can set it in the Authorization Header field of the App Repository form. Note that in this case it will be prefixed with `Bearer`. For example: `Bearer UVd4aFpHUnBianB2Y0dWdUlIT=`.
 
+## Harbor
+
+[Harbor](https://github.com/goharbor/harbor) is an open source trusted cloud native registry project that stores, signs, and scans content, e.g. Docker images. Harbor is hosted by the [Cloud Native Computing Foundation](https://cncf.io/). Since version 1.6.0, Harbor is a composite cloud native registry which supports both container image management and Helm chart management. Harbor integrates [ChartMuseum](https://chartmuseum.com) to provide the Helm chart repository functionality. The access to Helm Charts in a Harbor Chart Repository can be controlled via Role-Based Access Control.
+
+To use Harbor with Kubeapps, first deploy Harbor using [Harbor offline installer](https://github.com/goharbor/harbor/blob/master/docs/installation_guide.md#downloading-the-installer). Here are the minimum steps required on a Linux machine to install Harbor for serving as Helm Chart Repository.
+```
+$ wget https://storage.googleapis.com/harbor-releases/release-1.8.0/harbor-offline-installer-v1.8.1.tgz
+$ tar xvf harbor-offline-installer-v1.8.1.tgz
+$ cd harbor
+$ sed -i 's/hostname: reg.mydomain.com/hostname: <Current-Machine-IP>/' harbor.yml
```

Comment:
can you explain how you can get this `Current-Machine-IP`?

Reply:
Since "Here are the minimum steps required on a Linux machine to install Harbor", the `Current-Machine-IP` means the IP of this Linux machine. We can run ifconfig to get its IP.